edx_proctoring/static/proctoring/templates/add-new-allowance.underscore

```diff
@@ -2,39 +2,80 @@
     select#proctored_exam {
         width: 100%;
     }
+    select#timed_exam {
```

Comment:
You would need a new template underscore file. Copy everything here into the new one. Revert the changes you have on this underscore file to the original.